test/fixtures/forks/zcash/transaction_builder_custom.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
It is not clear to me, why this part of the fixtures was separated out. Is this in anticipation of another change? Is this code that is special to Zcash whereas the other is generic to the UTXO coins and could be combined in some fashion? Could you please add a comment to the commit message or explain in some other way?

Reply:
the tests here are in a different format than the one expected in `test/transaction_builder.js`. With `mergeFixtures` these tests would be merged with the base bitcoin `transaction_builder.js` tests. I'll split out this change into a separate commit.